mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/model/spi/FieldPropertyHandle.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.model.spi;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Field;
+
+import org.hibernate.search.mapper.pojo.logging.impl.Log;
+import org.hibernate.search.util.common.logging.impl.LoggerFactory;
+
+/**
+ * @author Yoann Rodiere
+ */
+public final class FieldPropertyHandle implements PropertyHandle {
```

Comment:
Why are they both in `spi`?

Reply:
Because they are used from another module: the ORM mapper.